templates/primary-secondary/primary-secondary.js

```diff
@@ -26,6 +28,9 @@ class TemplatePrimarySecondary extends LitElement {
 				"primary divider secondary"
 				"footer footer footer";
 				height: 100vh;
+				margin-left: auto;
+				margin-right: auto;
+				min-width: min-content;
 			}
```

Comment:
You should be able to use a selector just like with navigation instead of doing custom stuff in `render()`:

```css
:host([width-type="normal"]) {
    max-width: 1230px;
}
```